packages/events/src/router/event/event.actions.correction.test.ts

```diff
@@ -84,6 +84,75 @@ test(`${ActionType.APPROVE_CORRECTION} validation error message contains all th
   ).rejects.matchSnapshot()
 })
 
+test(`${ActionType.REQUEST_CORRECTION} Allows passing fields that are conditionally removed`, async () => {
```

Comment:
Could you reword this a bit? In the snapshot I'm seeing which made me think this was about ensuring invalid values return an error

```
exports[`REQUEST_CORRECTION Allows passing fields that are conditionally removed 1`] = `[TRPCError: [{"message":"Please enter a valid date","id":"applicant.dob","value":"02-1-2024"},{"message":"Invalid date. Please use the format YYYY-MM-DD","id":"applicant.dob","value":"02-1-2024"}]]`;
```